app/controllers/api/v1/admin/user/evaluations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Api::V1::Admin::User::EvaluationsController < Api::V1::BaseController
+  before_action :authenticate_user!
+  before_action :load_user, only: [:index, :create]
+  before_action :load_evaluation, only: [:update, :destroy]
+
+  def index
+    @evaluations = @user.evaluations
+    authorize @evaluations
+    respond_with @evaluations
+  end
+
+  def show
+    @evaluation = Evaluation.includes(:goals).find(params[:id])
+    authorize @evaluation
+    respond_with @evaluation, json: { evaluation: @evaluation,
+                                      goals: @evaluation.goals }
+  end
+
+  def create
+    @evaluation = Evaluation.new(evaluation_params)
```

Comment:
build evaluation for user. In this case, you don't need to pass `user_id` from UI